src/dmd/backend/cg.d

```diff
@@ -52,3 +50,110 @@ Symbol *tls_get_addr_sym;       // function __tls_get_addr
+    } ();
+
+
+    ubyte[FLMAX] stackfl =
+    () {
+        ubyte[FLMAX] stackfl;
+        foreach (fl; [ FLauto,FLfast,FLpara,FLcs,FLfltreg,FLallocatmp,FLbprel,FLstack,FLregsave,
+                       FLfuncarg,
+                       FLndp,
+                     ])
+        {
+            stackfl[fl] = 1;
+        }
+        return stackfl;
+    }();
+
+    ubyte[FLMAX] segfl =
+    () {
+        ubyte[FLMAX] segfl;
+
+        /* Segment registers    */
+        enum ES = 0;
+        enum CS = 1;
+        enum SS = 2;
+        enum DS = 3;
+
+        foreach (i;  0 .. FLMAX)
+        {   switch (i)
+            {
+                case 0:         segfl[i] = cast(byte)-1;  break;
+                case FLconst:   segfl[i] = cast(byte)-1;  break;
+                case FLoper:    segfl[i] = cast(byte)-1;  break;
+                case FLfunc:    segfl[i] = CS;  break;
+                case FLdata:    segfl[i] = DS;  break;
+                case FLudata:   segfl[i] = DS;  break;
+                case FLreg:     segfl[i] = cast(byte)-1;  break;
+                case FLpseudo:  segfl[i] = cast(byte)-1;  break;
+                case FLauto:    segfl[i] = SS;  break;
+                case FLfast:    segfl[i] = SS;  break;
+                case FLstack:   segfl[i] = SS;  break;
+                case FLbprel:   segfl[i] = SS;  break;
+                case FLpara:    segfl[i] = SS;  break;
+                case FLextern:  segfl[i] = DS;  break;
+                case FLcode:    segfl[i] = CS;  break;
+                case FLblock:   segfl[i] = CS;  break;
+                case FLblockoff: segfl[i] = CS; break;
+                case FLcs:      segfl[i] = SS;  break;
+                case FLregsave: segfl[i] = SS;  break;
+                case FLndp:     segfl[i] = SS;  break;
+                case FLswitch:  segfl[i] = cast(byte)-1;  break;
+                case FLfltreg:  segfl[i] = SS;  break;
+                case FLoffset:  segfl[i] = cast(byte)-1;  break;
+                case FLfardata: segfl[i] = cast(byte)-1;  break;
+                case FLcsdata:  segfl[i] = CS;  break;
+                case FLdatseg:  segfl[i] = DS;  break;
+                case FLctor:    segfl[i] = cast(byte)-1;  break;
+                case FLdtor:    segfl[i] = cast(byte)-1;  break;
+                case FLdsymbol: segfl[i] = cast(byte)-1;  break;
+                case FLgot:     segfl[i] = cast(byte)-1;  break;
+                case FLgotoff:  segfl[i] = cast(byte)-1;  break;
+                case FLlocalsize: segfl[i] = cast(byte)-1;        break;
+                case FLtlsdata: segfl[i] = cast(byte)-1;  break;
+                case FLframehandler:    segfl[i] = cast(byte)-1;  break;
+                case FLasm:     segfl[i] = cast(byte)-1;  break;
+                case FLallocatmp:       segfl[i] = SS;  break;
+                case FLfuncarg:         segfl[i] = SS;  break;
+                default:
+                        printf("error in segfl[%d]\n", i);
```

Comment:
Is that CTFE-able ?

Reply:
No, but that code is unreachable.

Reply:
Then we can just remove the `printf` and the `import`